worldedit-core/src/main/java/com/sk89q/worldedit/extension/factory/parser/DefaultBlockParser.java

```diff
@@ -206,7 +206,10 @@ private static Map<Property<?>, Object> parseProperties(BlockType type, String[]
     public Stream<String> getSuggestions(String input, ParserContext context) {
         final int idx = input.lastIndexOf('[');
         if (idx < 0) {
-            return SuggestionHelper.getNamespacedRegistrySuggestions(BlockType.REGISTRY, input);
+            return Stream.concat(
+                Stream.of("hand", "offhand", "pos1"),
```

Comment:
Shouldn't hand/offhand only be suggested if the actor is a player? This can be called in a console context so I don't think you can assume it's a player.